[MCKIN-8184] Update the course enrollment list api endpoint docstring

lgp171188 opened the pull request:
Also update the docstring of functions used by the endpoint. This is a follow-up PR to the https://github.com/edx-solutions/edx-platform/pull/1142 PR with fixes to the docstring.

**Reviewers**
- [ ] @pomegranited
- [ ] @xitij2000

pomegranited commented on common/djangoapps/enrollment/views.py:
```diff
@@ -712,10 +712,12 @@ class CourseEnrollmentsApiListView(DeveloperErrorViewMixin, ListAPIView):
         **Use Cases**
 
-            Get a list of all course enrollments given a course ID or list of usernames.
+            Get a list of all course enrollments, optionally filtered by a course ID or list of usernames.
```
nit: "optionally filtered by a course ID *and/or* list of usernames"

I know.. programmatically "and/or" is the same as "or", but to humans, I think this is clearer.

pomegranited replied:
👍

pomegranited commented on common/djangoapps/enrollment/views.py:
```diff
@@ -725,15 +727,14 @@ class CourseEnrollmentsApiListView(DeveloperErrorViewMixin, ListAPIView):
+              of the given users. Optional.
 
-            * page_size: Number of results to return per page.
+            * page_size: Number of results to return per page. Optional.
```
Could you indicate the default page size if this parameter is not passed?  Max page size too?

pomegranited replied:
👍

pomegranited commented on common/djangoapps/enrollment/views.py:
```diff
@@ -725,15 +727,14 @@ class CourseEnrollmentsApiListView(DeveloperErrorViewMixin, ListAPIView):
+            * page_size: Number of results to return per page. Optional.
 
-            * page: Page number to retrieve.
+            * page: Page number to retrieve. Optional.
```
Could you indicate the default page number (1)?

pomegranited replied:
👍

lgp171188 commented:
@pomegranited, I have addressed your review comments in https://github.com/edx-solutions/edx-platform/pull/1146/commits/da094934690576b6858ddd6ff93bcf7c862bed87
